### PR TITLE
[IMP] stock_available_to_promise_release: handle the priority

### PR DESCRIPTION
**Pull request by sebalix**
Ref. 1726

### CODE REVIEW

**Review by jgrandguillaume — APPROVED** [review]
All good thx !

File: stock_available_to_promise_release/models/stock_move.py [hunk 1]
```diff
@@ -93,7 +93,12 @@ def _previous_promised_quantity_domain(self):
         # and priority is higher than the current one
         domain_not_released = [
             ("need_release", "=", True),
+            # ("date_priority", "<", self.date_priority),
```
Comment (simahawk) [inline stock_available_to_promise_release/models/stock_move.py]: comment to remove?

**Review by simahawk — APPROVED** [review]
just code review, LG

File: stock_available_to_promise_release/models/stock_move.py (hunk 1 above)
Comment (sebalix) [reply stock_available_to_promise_release/models/stock_move.py]: fixed

**Pull request merged**